.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: ❓ Janus IDP Community Slack
-    url: https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg
-    about: Please ask and answer questions here.
+  - name: ❓ Red Hat Developer Hub JIRA
+    url: https://issues.redhat.com/browse/RHIDP
+    about: Please report any issues or feature requests in JIRA - https://issues.redhat.com/browse/RHIDP
```

Comment:
Can you post a message saying "effective on <date>" and explain that all open GH issues will be converted to JIRAs.  We should add the JIRA links to the GH issues so reporters can easily find them

Reply:
I'm not planning to manually convert 63 showcase and 112 plugins repo issues to JIRAs.

Reply:
As I mentioned, we are doing the community a disservice by abandoning the open issues.  The least we can do is scrub through the list and address the ones we have no plans to fix so there's closure.  For the record, many of the opened issues have been sync'd to JIRA so really, it's [36 showcase](https://github.com/janus-idp/backstage-showcase/issues?q=is%3Aopen+is%3Aissue+-label%3Ajira+) and [76 plugins](https://github.com/janus-idp/backstage-plugins/issues?q=is%3Aopen+is%3Aissue+-label%3Ajira+).  We will also need to get input from the @janus-idp/devex-uxe , @janus-idp/orchestrator , @janus-idp/kiali  @janus-idp/rhtap teams since some of the issues may be opened against their plugins

Reply:
OK, here's a JIRA for that. https://issues.redhat.com/browse/RHIDP-3690